From e813c4189d6cead5865d7017c21aa68da8b68acc Mon Sep 17 00:00:00 2001
From: Sejal Choudhary <sejalc230@gmail.com>
Date: Sat, 9 May 2020 14:52:13 +0530
Subject: [PATCH] Create Patch File

---
 src/DataManipulator.ts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/DataManipulator.ts b/src/DataManipulator.ts
index 6ed41c7..73d9284 100644
--- a/src/DataManipulator.ts
+++ b/src/DataManipulator.ts
@@ -12,9 +12,9 @@ export interface Row {
 
 
 export class DataManipulator {
-  static generateRow(serverResponds: ServerRespond[]): Row {
-      const priceABC=(serverResponds[0].top_ask.price + serverResponds[0].top_bid.price)/2;
-      const priceDEF=(serverResponds[1].top_ask.price + serverResponds[1].top_bid.price)/2;
+  static generateRow(serverRespond: ServerRespond[]): Row {
+      const priceABC=(serverRespond[0].top_ask.price + serverRespond[0].top_bid.price)/2;
+      const priceDEF=(serverRespond[1].top_ask.price + serverRespond[1].top_bid.price)/2;
       const ratio=priceABC/priceDEF;
       const upperBound=1+0.05;
       const lowerBound=1-0.05;
@@ -22,7 +22,7 @@ export class DataManipulator {
         price_abc:priceABC,
         price_def:priceDEF,
         ratio,
-        timestamp:serverResponds[0].timestamp>serverResponds[1].timestamp?serverResponds[0].timestamp:serverResponds[1].timestamp,
+        timestamp:serverRespond[0].timestamp>serverRespond[1].timestamp?serverRespond[0].timestamp:serverRespond[1].timestamp,
         upper_bound:upperBound,
         lower_bound:lowerBound,
         trigger_alert:(ratio>upperBound || ratio < lowerBound)? ratio :undefined,
-- 
2.25.1.windows.1

